arch/arm/soc/rda_5981/Kconfig

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright (c) 2017 RDA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_FAMILY_RDA5981
```

Comment:
There is not yet a more generic name. 5981 is the SoC  famliy name, which includes 5981a now.